### PR TITLE
[zh-cn] sync concepts/workloads/*

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed:
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/cron-jobs.md
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/daemonset.md
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/deployment.md
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/job.md
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/replicaset.md
#       modified:   content/zh-cn/docs/concepts/workloads/controllers/replicationcontroller.md
#       modified:   content/zh-cn/docs/concepts/workloads/pods/_index.md
#       modified:   content/zh-cn/docs/concepts/workloads/pods/init-containers.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -207,15 +205,33 @@ that updates those files from a remote source, as in the following diagram:
...
+Enabling the `SidecarContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+allows you to specify `restartPolicy: Always` for init containers.
+Setting the `Always` restart policy ensures that the init containers where you set it are
+kept running during the entire lifetime of the Pod.
+See [Sidecar containers and restartPolicy](/docs/concepts/workloads/pods/init-containers/#sidecar-containers-and-restartpolicy)
+for more details.
+-->
+启用 `SidecarContainers` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)允许你为
+init 容器指定 `restartPolicy: Always`。
+在 Pod 的整个生命周期内保持运行。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/pods/_index.md:228]:
```suggestion
设置重启策略为 `Always` 会确保 Init 容器在 Pod 的整个生命周期内保持运行。
```

File: content/zh-cn/docs/concepts/workloads/pods/_index.md [hunk 2]
```diff
@@ -207,15 +205,32 @@ that updates those files from a remote source, as in the following diagram:
...
+<!--
+Enabling the `SidecarContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+allows you to specify `restartPolicy: Always` for init containers.
+Setting the `Always` restart policy ensures that the init containers where you set it are
+kept running during the entire lifetime of the Pod.
+See [Sidecar containers and restartPolicy](/docs/concepts/workloads/pods/init-containers/#sidecar-containers-and-restartpolicy)
+for more details.
+-->
+启用 `SidecarContainers` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)允许你为
+设置重启策略为 `Always` 会确保 Init 容器在 Pod 的整个生命周期内保持运行。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/_index.md:227]:
```suggestion
启用 `SidecarContainers` [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)允许你为
Init 容器指定 `restartPolicy: Always`。设置重启策略为 `Always` 会确保 Init 容器在 Pod 的整个生命周期内保持运行。
```

**Review by ydFu — APPROVED** [review]
/lgtm 
/approve

**Review by Arhell — COMMENTED** [review]
Please re-puch commit to restart netlify

**Pull request merged**